entries/o/o2switch.fr.json

```diff
@@ -0,0 +1,14 @@
+{
+    "o2switch": {
+        "domain":"o2switch.fr",
+        "url": "https://clients.o2switch.fr",

```

Comment:
```suggestion
```

The `url` key is not needed because you can access the base domain (`https://o2switch.fr`)